nestdict/change_value.py

```diff
@@ -0,0 +1,62 @@
+"""# implementation of change_value function
+
+    * it changes the value of a particular key and returns the dictionary back.
+    * it also works with nested dictionary.
+"""
+    
+def change_value(obj,*args,value):
+    
+    """This function accept three params
+       and iterats over the obj(dict) and replace value
+       of the key.
+       
+       Arg:
+       
+         obj (dict) = dictionary object.
+         *args (list) = must pass the keys in a list.
+         value = value to replace insitited of old value
+         
+       Note: *args must be in correct order.
+       
+       Use Case:
+             dic = {
+                   "cars_owned :{
+                   "sedan":1,
+                   "suv":3
+                   },
+                }
+                
+             print("original :", dic)
+             print(nestdict.change_value(dic,["cars_owned","suv"],value = 100))
+             
+             #the elements inside the list must follow the correct order.
+             
+       output:
+       
+       original : {"cars_owned":{"sedan":1,"suv":3},}
+       
+       {"cars_owned":{"sedan":1,"suv":100},}
+       
+       
+         
+      for more understanding read README.md
+      """
+
+    changed_obj = obj
+    
+    keys = list(*args)
+    
+    if len(keys)>1:
+        for element in keys:
+            obj = obj[element]

```

Comment:
return None or original obj if element not found in obj.
current code will throw key error.

Reply:
Solved key error problem.

Reply:
Solved key error problem. It will return None if element not found.

Reply:
It returns the original obj if element not found.